tests/runfiles/runfiles_test.py

```diff
@@ -126,23 +131,25 @@ def testManifestBasedRunfilesEnvVars(self):
                 },
             )
 
-    def testCreatesDirectoryBasedRunfiles(self):
+    def testCreatesDirectoryBasedRunfiles(self) -> None:
         r = runfiles.Create(
             {
                 "RUNFILES_DIR": "runfiles/dir",
                 "TEST_SRCDIR": "always ignored",
             }
         )
+        assert r is not None
```

Comment:
I originally had this but it mypy would immediately complain since it doesn't seem to understand the API. I could do this and then use a `cast` but I felt this was the lesser evil. I don't really care which it is though so with that said, which would you prefer? or is there a better path?

Reply:
SGTM to leave the `assert r is not None` but it may be nice to add a comment as to why we are not using the `unittest` API.

Reply:
Done!